### DISCUSSION TITLE
Something is wrong after second login of user via LDAP

### DISCUSSION THREAD

**Discussion by spmml** [kimai/kimai#2419] [General]
The whole installation worked well. But when a user logged in via LDAP and logged out, a new login is not possible.
The following error occurs: "Something is wrong
A critical error occurred, please try again. You may have found a software problem, please contact your administrator if the problem persists."

What's causing this error? My local.yaml has just modified fields of the sample configuration for my LDAP  at the documentation. 

var/log/prod.log is attached
[prod.log](https://github.com/kevinpapst/kimai2/files/6118526/prod.log)

**Comment by kevinpapst** [comment]
Misconfigured LDAP. 
Kimai tried to re-create the user, even though it already exists in the database.
I assume the username attribute is wrong.
